scripts/create-all-topics.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+kafka_ready() {
+    kafka-topics.sh --bootstrap-server localhost:9092 --list > /dev/null 2>&1
+}
+
+# Wait for Kafka to be ready
+while ! kafka_ready; do
+    echo "Waiting for Kafka to be ready..."
+    sleep 5
+done
+
+echo "Creating topics now"
+
+kafka-topics.sh --create --bootstrap-server localhost:9092 --topic flow --if-not-exists
```

Comment:
Do these in order and create them all:

channel
language
flow
retriever
indexer